### ISSUE TITLE
Very strange error on Windows: No module named'PySide'

### ISSUE THREAD

**Issue by KaneGreen** [jupyter/qtconsole#515]
I often use qtconsole, so I set the following command as a Windows shortcut (.lnk):
```
C:\ProgramData\Anaconda3\pythonw.exe C:\ProgramData\Anaconda3\cwp.py C:\ProgramData\Anaconda3 C:\ProgramData\Anaconda3\pythonw.exe C:\ProgramData\Anaconda3\Scripts\jupyter-qtconsole-script.py
```
However, when I double-click this shortcut, qtconsole fails to start, prompting:
```
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtpy\__init__.py", line 209, in <module>
    from PySide import __version__ as PYSIDE_VERSION  # analysis:ignore
ModuleNotFoundError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\Scripts\jupyter-qtconsole-script.py", line 5, in <module>
    from qtconsole.qtconsoleapp import main
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\qtconsoleapp.py", line 61, in <module>
    from qtpy import QtCore, QtGui, QtWidgets, QT_VERSION
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtpy\__init__.py", line 215, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found
```

❌ I tried to run this command in CMD and it failed.
✔ I started CMD in Windows Terminal, and then run the command to successfully start qtconsole.

❌I cannot start qtconsole in the "Run window" (Win+R).
✔I directly used the Ctrl+V key combination to paste the command in the start menu and then pressed Enter. This "Windows Search" could successfully start qtconsole.

System: Windows 10 21H2
qtconsole version: 5.1.1
Python version: 3.9.7

**Comment by dalthviz** [comment]
Hi @KaneGreen just in case, what is the QtPy version do you have installed?

**Comment by KaneGreen** [comment]
> Hi @KaneGreen just in case, what is the QtPy version do you have installed?

5.9.2

**Comment by dalthviz** [comment]
After checking a little bit I think the issue is related with the context the commands are run (to actually run succesfully you need a cmd context). I think something like this could work:

    C:\Windows\System32\cmd.exe /c "C:\ProgramData\Anaconda3\pythonw.exe C:\ProgramData\Anaconda3\cwp.py C:\ProgramData\Anaconda3 C:\ProgramData\Anaconda3\pythonw.exe C:\ProgramData\Anaconda3\Scripts\jupyter-qtconsole-script.py"

So basically the shortcut will open a cmd instance and use it to run the command. Could you try something like that @KaneGreen ?

**Comment by KaneGreen** [comment]
> After checking a little bit I think the issue is related with the context the commands are run (to actually run succesfully you need a cmd context). I think something like this could work:
> 
> ```
> C:\Windows\System32\cmd.exe /c "C:\ProgramData\Anaconda3\pythonw.exe C:\ProgramData\Anaconda3\cwp.py C:\ProgramData\Anaconda3 C:\ProgramData\Anaconda3\pythonw.exe C:\ProgramData\Anaconda3\Scripts\jupyter-qtconsole-script.py"
> ```
> 
> So basically the shortcut will open a cmd instance and use it to run the command. Could you try something like that @KaneGreen ?

It works!
But, after I tried this, I can run it without `C:\Windows\System32\cmd.exe /c `. That's really weird. 
I think this problem is solved and I will close this issue.

Thank you for your concern.
